FirebaseInstallations/Source/Library/FIRInstallationsItem.h

```diff
@@ -24,13 +24,19 @@ typedef NS_ENUM(NSInteger, FIRInstallationsStatus);
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ *  The class represents the required installation ID and auth token data including possible states.
+ *  The data is stored to Keychain via `FIRInstallationsStoredItem` which has only the storage
+ *  relevant data and does not contain any logic. `FIRInstallationsItem` must be used on the logic
+ * level (not `FIRInstallationsStoredItem`).
```

Comment:
Nit: indentation

Reply:
uh... ./scripts/style.sh messes up the indentation on the comments :(